lib/runner.js

```diff
@@ -158,14 +158,15 @@ Runner.prototype._addTestResult = function (test) {
 		this.stats.failCount++;
 	}
 
-	this.results.push({
+	var props = {
 		duration: test.duration,
 		title: test.title,
-		error: test.assertError,
+		err: test.assertError,
```

Comment:
I feel it's different. This is more like passing around data than variables. Think of it as a table of data and the keys are the headers. At least how it's like in my mind.